console-extensions.json

```diff
@@ -357,5 +357,46 @@
       },
       "component": { "$codeRef": "MigrationPoliciesList" }
     }
+  },
+  {
+    "type": "dev-console.add/action",
+    "properties": {
+      "id": "dev-catalog-virtualization",
+      "groupId": "developer-catalog",
+      "href": "/k8s/ns/:namespace/templatescatalog",
```

Comment:
The Add VM action from the dev perspective now leads to the new catalog.
What should we do with the old dev form,catalog @yaacov ?

Reply:
AFAU the catalog is part of the admin perspective ? is it ok to send users from the dev perspective to the admin one ?
will it work for users that can only see the dev perspective ?